common/src/main/java/org/corfudb/common/metrics/micrometer/README.md

```diff
@@ -13,63 +13,89 @@ Logger name should be "org.corfudb.client.metricsdata".
 *   **logreplication.sender.duration.seconds**: Duration of sending a log entry in seconds (throughput, mean and max), distinguished by replication type (snapshot, logentry) and status (success, failure).
 *   **logreplication.rtt.seconds**: Duration of sending a message overall (throughput, mean and max).
 *   **logreplication.snapshot.completed.count**: Number of snapshot syncs completed.
-*   **logreplication.snapshot.duration**: Duration of completing a snapshot sync in seconds (throughput, mean and max).
+*   **logreplication.snapshot.duration.seconds**: Duration of completing a snapshot sync in seconds (throughput, mean and max).
 *   **logreplication.acks**: Number of acks, distinguished by replication type (snapshot, logentry).
 *   **logreplication.messages**: Number of messages sent, distinguished by replication type (snapshot, logentry).
-*   **logreplication.opaque.count_valid**: Number of opaque entries per message (rate, mean, max).
-*   **logreplication.opaque.count_total**: Number of overall opaque entries (rate, mean, max).
-*   **logreplication.opaque.count_valid**: Number of valid opaque entries (rate, mean, max).
+*   **logreplication.opaque.count_per_message**: Number of opaque entries per message (rate, mean, max).
+*   **logreplication.opaque.count\_total**: Number of overall opaque entries (rate, mean, max).
```

Comment:
why do you escape the underscore here but not in line 19?

Reply:
it's generate by the linter tool to pass the static code check in github